charts/opentelemetry-operator/templates/serviceaccount.yaml

```diff
@@ -2,8 +2,13 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  labels:
-    app.kubernetes.io/name: opentelemetry-operator
   name: {{ template "opentelemetry-operator.serviceAccountName" . }}-controller-manager
   namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "opentelemetry-operator.labels" . | nindent 4 }}
+    {{- include "opentelemetry-operator.selectorLabels" . | nindent 4 }}
```

Comment:
This looks like something unrelated to this PR. I don't believe we have these labels consistently applied to other resources, so it should be handled separately.

Reply:
sure, i can remove this section. Figured while I was there i should give it what the expectation of the consumer would be.